From a2df580982a2c364aa35da82aa66feddf2e56f36 Mon Sep 17 00:00:00 2001
From: Christian Hewitt <christianshewitt@gmail.com>
Date: Mon, 1 Feb 2021 19:27:40 +0000
Subject: [PATCH 008/119] LOCAL: arm64: dts: meson: add rtc/vrtc aliases to
 Minix NEO U9-H

Add node aliases to prevent meson-vrtc from claiming /dev/rtc0

Signed-off-by: Christian Hewitt <christianshewitt@gmail.com>
---
 arch/arm64/boot/dts/amlogic/meson-gxm-minix-neo-u9h.dts | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/arch/arm64/boot/dts/amlogic/meson-gxm-minix-neo-u9h.dts b/arch/arm64/boot/dts/amlogic/meson-gxm-minix-neo-u9h.dts
index b8ef3bd8b840..0920e7b096f5 100644
--- a/arch/arm64/boot/dts/amlogic/meson-gxm-minix-neo-u9h.dts
+++ b/arch/arm64/boot/dts/amlogic/meson-gxm-minix-neo-u9h.dts
@@ -14,6 +14,11 @@ / {
 	compatible = "minix,neo-u9h", "amlogic,s912", "amlogic,meson-gxm";
 	model = "Minix Neo U9-H";
 
+	aliases {
+		rtc0 = &rtc;
+		rtc1 = &vrtc;
+	};
+
 	leds {
 		compatible = "gpio-leds";
 
-- 
2.34.1

